src/Polly.RateLimiting/RateLimiterResilienceStrategy.cs

```diff
@@ -5,6 +5,7 @@ namespace Polly.RateLimiting;
 
 internal sealed class RateLimiterResilienceStrategy : ResilienceStrategy, IDisposable, IAsyncDisposable
 {
+    private const string Message = "The operation could not be executed because it was rejected by the rate limiter.";
```

Comment:
As it's only used in one method, I think it would be better to declare this constant inline in the method body.